castor.php

```diff
@@ -96,7 +96,7 @@ function cache_clear(): void
 {
     // io()->title('Clearing the application cache');
 
-    // docker_compose_run('rm -rf var/cache/ && bin/console cache:warmup');
+    // docker_compose_run('sh -c "rm -rf var/cache/ && bin/console cache:warmup"');
```

Comment:
I would split the lines in two lines. 
**But**, I never noticed that, and I think the issue should be fixed on the other side